backend.c

```diff
@@ -1458,16 +1458,17 @@ static bool keep_running(struct thread_data *td)
 	return false;
 }
 
-static int exec_string(struct thread_options *o, const char *string, const char *mode)
+static int exec_string(struct thread_options *o, const char *string,
+		       const char *mode)
```

Comment:
Can't tell if this indenting is correct (might just be github)

Reply:
In my editor indentation looks correct. Maybe github has trouble with displaying tabs correctly?